packages/react/src/components/ComposedModal/ComposedModal.js

```diff
@@ -22,6 +22,7 @@ export default class ComposedModal extends Component {
   state = {};
 
   static defaultProps = {
+    ariaLabel: 'modal-container',
```

Comment:
Definitely a bummer that we can't make this required without breaking people, would there be a good prop that we could derive a meaningful label from, potentially?

Reply:
I originally didn't think we'd be able to, but after taking a deeper look I've modified the code a bit so that it will only generate one if a user does not specify their own `aria-label` and will be based on the `label`, matching `Modal` functionality